### ISSUE TITLE
Should datasets have a protocols section?

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/portal-ui#1721]
Looking at [a dataset](https://portal-prod.stage.hubmapconsortium.org/browse/dataset/3d2c9295c0e05f4d81d23ee95e9dad23#metadata), there are protocol DOIs in the metadata, but there is not a protocols section in the UI, as there is with [this Sample](https://portal-prod.stage.hubmapconsortium.org/browse/sample/913de60dd820bd3e42fd1515b096b0d8#protocols).

My guess is that this is just an oversight, and there should be a protocols section for Datasets as well as for Samples? I think the datastructures involved may be a little different: For Samples, when there are multiple protocols, I don't think there's anything in the Entity JSON to distinguish them: For Datasets, they would be pulled from the metadata, so we have a field name and a description.

**Comment by ngehlenborg** [comment]
Yes, we should include a protocols section if the criteria that are being discussed in #2034 apply.
